fluentd/Dockerfile

```diff
@@ -23,7 +23,6 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
   mkdir -p ${HOME} && \
   gem install --no-ri fluentd fluent-plugin-kubernetes_metadata_filter fluent-plugin-elasticsearch fluent-plugin-flatten-hash
 
-ADD out_elasticsearch*.rb /opt/app-root/src/gems/fluent-plugin-elasticsearch-1.0.0/lib/fluent/plugin/
```

Comment:
@sosiouxme yeah, the latest release of fluent-plugin-elasticsearch contains these changes now, we were already installing it in the line above but then overlaying the files we were providing